PluralKit.Bot/Services/EmbedService.cs

```diff
@@ -301,6 +301,8 @@ public async Task<Embed> CreateMessageInfoEmbed(FullMessage msg)
                 if (member != null)
                     // Don't do an extra request if we already have this info from the member lookup
                     userInfo = member.User;
+                else userInfo = await _cache.GetOrFetchUser(_rest, msg.Message.Sender);
+
                 memberInfo = member;
             }
             else userInfo = await _cache.GetOrFetchUser(_rest, msg.Message.Sender);
```

Comment:
as to not duplicate code, would it be possible to `if (userInfo == null) userInfo = await ...` outside of the `if (channel != null)` block?